planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/util/YAMLmatchmakerToOptimizerParser.java

```diff
@@ -39,18 +40,18 @@
 
    // Reducing verbosity . If somebody knows a better approach for doing this (I
    // could not set dynamically the level of the logging) it should be changed
-   private static int           BeeingTooVerboseWithLackOfInformationInCloudOffers = 3;
+   private static int BeeingTooVerboseWithLackOfInformationInCloudOffers = 3;
 
    static Logger log = LoggerFactory.getLogger(YAMLmatchmakerToOptimizerParser.class);
 
    @SuppressWarnings("unchecked")
    public static List<Object> getListofOptions(String appModel) {
       Yaml yamlApp = new Yaml();
-      if(log.isDebugEnabled()){
-      log.debug("Loading String to a YAML using sknakeyaml.Yaml");
+      if (log.isDebugEnabled()) {
+         log.debug("Loading String to a YAML using sknakeyaml.Yaml");
```

Comment:
sl4fj adds a nice feature to avoid the concatenation of strings and the need to use isXXXEnabled() to improve the performance. Instead of using:
```log.debug("responsetime = " + responsetime + " ms");```

You can use:
```log.debug("responsetime = {} ms", responsetime);

The substitution is made internally only if the log is efective :)